components/map/Map.js

```diff
@@ -0,0 +1,34 @@
+import { MapContainer, TileLayer } from "react-leaflet";
+
+import "leaflet/dist/leaflet.css";
+import styled from "styled-components";
+
+export default function Map() {
+  return (
+    <>
+      <MapContainer
+        center={[58.034125450605316, 7.454502477686363]}
+        zoom={11}
+        scrollWheelZoom={true}
+        onClick={(event) => getCoord(event.latlng)}
```

Comment:
Just curious, where does the getCoord function comes from? I can't seem to find it. Does it work?

Reply:
Ah, i forgot to delete that one. I tried to get the coords with event.latlng but unfortunately it did not work :(